benches/benches.rs

```diff
@@ -0,0 +1,29 @@
+#![allow(dead_code)] // Due to criterion we need this to avoid warnings.
+
+extern crate criterion;
+extern crate env_logger;
+extern crate raft;
+
+use criterion::Criterion;
+use std::time::Duration;
+
+mod suites;
+
+pub const DEFAULT_RAFT_SETS: [(usize, usize); 4] = [(0, 0), (3, 1), (5, 2), (7, 3)];
+
+fn main() {
+    criterion::init_logging();
+    let _ = env_logger::try_init();
```

Comment:
Seems to be duplicated? criterion::init_logging will use a SimpleLogger to initialize `log`.

Reply:
Oh nice catch. :)